src/axom/klee/Geometry.cpp

```diff
@@ -38,5 +145,71 @@ TransformableGeometryProperties Geometry::getEndProperties() const
   return m_startProperties;
 }
 
+const axom::sidre::Group* Geometry::getBlueprintMesh() const
+{
+  SLIC_ASSERT_MSG(
+    m_meshGroup,
+    axom::fmt::format(
+      "The Geometry format '{}' is not specified "
+      "as a blueprint mesh and/or has not been converted into one.",
+      m_format));
+  return m_meshGroup;
+}
+
+const std::string& Geometry::getBlueprintTopology() const
+{
+  SLIC_ASSERT_MSG(
+    m_meshGroup,
+    axom::fmt::format(
+      "The Geometry format '{}' is not specified "
+      "as a blueprint mesh and/or has not been converted into one.",
+      m_format));
+  return m_topology;
+}
+
+#if 0
```

Comment:
Should this be removed if not used?

Reply:
It's temporarily disabled due to a weird crash.  Will fix.

Reply:
See issue https://github.com/LLNL/axom/issues/1445